testing/citest/spinnaker_testing/dcoscli_agent.py

```diff
@@ -0,0 +1,35 @@
+# Standard python modules.
```

Comment:
We can definitely look into contributing that to citest.  For now I'll move it another package in this repo.  I'm not very experienced with Python though and I'm running into a problem.  Right now it looks like:

```
.
├── citest
│   ├── __init__.py
│   └── dcos_testing
│       ├── __init__.py
│       ├── dcos_contract.py
│       └── dcoscli_agent.py
├── spinnaker_testing
├── tests
│   ├── dcos_smoke_test.py
```

`dcos_smoke_test.py` has the following imports:
```python
import citest.json_contract as jc
import citest.json_predicate as jp
import citest.service_testing as st
import citest.dcos_testing as dcos
```

The package in this repo looks like it's  shadowing the citest library because it reports this error when run: `ImportError: No module named json_contract` Is there a way to prevent that from happening?